sklearn_hierarchical/tests/fixtures.py

```diff
@@ -39,7 +39,7 @@ def make_class_hierarchy(n, n_intermediate=None, n_leaf=None):
         if n_leaf is None:
             n_leaf = n - 1
 
-        G = DiGraph(data=product((ROOT,), range(n_leaf)))
+        G = DiGraph(product((ROOT,), range(n_leaf)))
```

Comment:
networkx changed the DiGraph initialization parameters (instead of `data`, now `incoming_graph_data`). Removed the use of this keyword to allow for backward and forward compatibility.